nomad/structs/structs.go

```diff
@@ -1653,6 +1654,15 @@ func (sc *ServiceCheck) validate() error {
 		return fmt.Errorf("interval (%v) can not be lower than %v", sc.Interval, minCheckInterval)
 	}
 
+	switch sc.Status {
+	case "":
+	case "passing":
```

Comment:
Use the constants defined here - https://godoc.org/github.com/hashicorp/consul/api#pkg-constants